.github/workflows/build.yml

```diff
@@ -0,0 +1,138 @@
+name: Build, publish
+on:
+  push:
+    tags:
+    - 'v*'
+    branches:
+    - '**'
+
+jobs:
```

Comment:
These are lengthy mostly due to Actions' way of doing things. There is space for improvement (as in, de-duplication and some generalisation), but I have not attempted any due to time constraints.